### DISCUSSION TITLE
Removing minio bucket replication failing - RELEASE.2022-09-17T00-09-45Z

### DISCUSSION THREAD

**Discussion by saandeepd** [minio/minio#16005] [Q&A]
Hi, 

Hi Everyone, I am using Minio helm version RELEASE.2022-09-17T00-09-45Z,
I have configured the Minio bucket server replication. When I am trying to remove the replication rule on the bucket getting the below error. Please let me know how can I fix this
```
sh-4.4$ mc --config-dir /tmp/.mc admin bucket remote rm site1012/testminio1 --arn $bucket_arn
mc: <ERROR> Unable to remove remote target. This ARN is in use by an existing configuration.
Below is the debug log for the above command
```

Following is the debug log

`sh-4.4$ mc --config-dir /tmp/.mc admin bucket remote rm site1012/testminio1 --arn "arn:minio:replication::bd05b4b2-0071-4c20-86bf-5d8975d54ea0:testminio1" --debug
mc: <DEBUG> DELETE /minio/admin/v3/remove-remote-target?arn=arn%3Aminio%3Areplication%3A%3Abd05b4b2-0071-4c20-86bf-5d8975d54ea0%3Atestminio1&bucket=testminio1 HTTP/1.1
Host: minioapi-doc-uiv.apps.cb1012.tre.nsn-rdnet.net
User-Agent: MinIO (linux; amd64) madmin-go/0.0.1 mc/RELEASE.2022-09-16T09-16-47Z
Accept-Encoding: zstd,gzip
Authorization: AWS4-HMAC-SHA256 Credential=uivuser1/20221104//s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=**REDACTED**
X-Amz-Content-Sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
X-Amz-Date: 20221104T114211Z

mc: <DEBUG> HTTP/1.1 400 Bad Request
Content-Length: 260
Accept-Ranges: bytes
Content-Security-Policy: block-all-mixed-content
Content-Type: application/json
Date: Fri, 04 Nov 2022 11:42:11 GMT
Server: MinIO
Set-Cookie: b74668570af6e170cadd1ee8dd2a836a=1de0ec9f26d46530c55aa2705b7d913c; path=/; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Origin
Vary: Accept-Encoding
X-Amz-Request-Id: 17245EEC5C45DEA2
X-Content-Type-Options: nosniff
X-Xss-Protection: 1; mode=block

{"Code":"XMinioAdminRemoteRemoveDisallowed","Message":"This ARN is in use by an existing configuration","BucketName":"testminio1","Resource":"/minio/admin/v3/remove-remote-target","RequestId":"17245EEC5C45DEA2","HostId":"bf38187f-6dae-46e1-854c-0ebde868da60"}
mc: <DEBUG> TLS Certificate found: 
mc: <DEBUG>  >> Expires: 2024-08-25 08:49:04 +0000 UTC
mc: <DEBUG> TLS Certificate found: 
mc: <DEBUG>  >> Expires: 2024-08-25 08:49:04 +0000 UTC
mc: <DEBUG> Response Time:  7.204839ms

---------START-HTTP---------
DELETE /minio/admin/v3/remove-remote-target?arn=arn%3Aminio%3Areplication%3A%3Abd05b4b2-0071-4c20-86bf-5d8975d54ea0%3Atestminio1&bucket=testminio1 HTTP/1.1
Host: minioapi-doc-uiv.apps.cb1012.tre.nsn-rdnet.net
User-Agent: MinIO (linux; amd64) madmin-go/0.0.1 mc/RELEASE.2022-09-16T09-16-47Z
Accept-Encoding: zstd,gzip
Authorization: AWS4-HMAC-SHA256 Credential=uivuser1/20221104//s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=**REDACTED**
X-Amz-Content-Sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
X-Amz-Date: 20221104T114211Z

HTTP/1.1 400 Bad Request
Content-Length: 260
Accept-Ranges: bytes
Content-Security-Policy: block-all-mixed-content
Content-Type: application/json
Date: Fri, 04 Nov 2022 11:42:11 GMT
Server: MinIO
Set-Cookie: b74668570af6e170cadd1ee8dd2a836a=1de0ec9f26d46530c55aa2705b7d913c; path=/; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Origin
Vary: Accept-Encoding
X-Amz-Request-Id: 17245EEC5C45DEA2
X-Content-Type-Options: nosniff
X-Xss-Protection: 1; mode=block

{"Code":"XMinioAdminRemoteRemoveDisallowed","Message":"This ARN is in use by an existing configuration","BucketName":"testminio1","Resource":"/minio/admin/v3/remove-remote-target","RequestId":"17245EEC5C45DEA2","HostId":"bf38187f-6dae-46e1-854c-0ebde868da60"}
---------END-HTTP---------
mc: <ERROR> Unable to remove remote target. This ARN is in use by an existing configuration
 (1) admin-bucket-remote-rm.go:84 cmd.mainAdminBucketRemoteRemove(..) Tags: [site1012/testminio1]
 (0) admin-bucket-remote-rm.go:84 cmd.mainAdminBucketRemoteRemove(..)
 Release-Tag:RELEASE.2022-09-16T09-16-47Z | Commit:ad4441ef2b4b | Host:miniomc | OS:linux | Arch:amd64 | Lang:go1.18.6 | Mem:2.7 MB/22 MB | Heap:2.7 MB/12 MB.
`

**Comment by ravindk89** [comment]
> mc: <ERROR> Unable to remove remote target. This ARN is in use by an existing configuration.

Have you checked your buckets to see if you are using this ARN for an existing replication rule? Use [`mc replicate ls`](https://min.io/docs/minio/linux/reference/minio-mc/mc-replicate-ls.html)
